examples/conditional-send/conditional_send.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/io;
+
+public function main() {
+    boolean isDataReady = true;
+
+    worker w1 {
+        // A send action can be used in a conditional context.
+        if isDataReady {
+            10 -> w2;
+        }
+    }
+
+    worker w2 returns int|error:NoMessage {
+        // Since the send action is within a conditional context and 
+        // there is a possibility that the send action may not be executed,
+        // error:NoMessage type is added.
+        int|error:NoMessage a = <- w1;
+        return a;
+    }
+
+    worker w3 {
+        if isDataReady {
+            1 -> w4;
+        } else {
+            0 -> w4;
+        }
+    }
+
+    worker w4 returns int|error:NoMessage {
+        // Since send action is used in two instances within worker `w3`
+        // an alternate receive action is used here.
```

Comment:
Technically it is not mandatory to use an alternate-receive here. Hence, "can be used" may be more appropriate?
```suggestion
        // The send action is used in two instances within worker `w3`.
        // Therefore, an alternate receive action can be used to receive them.
```